AppCenter/AppCenter/Internals/Storage/MSDBStorage.m

```diff
@@ -110,6 +111,22 @@ + (void)setVersion:(NSUInteger)version inOpenedDatabase:(void *)db {
   [MSDBStorage executeNonSelectionQuery:query inOpenedDatabase:db];
 }
 
++ (void)enableAutoVacuumInOpenedDatabase:(void *)db {
+  NSArray<NSArray *> *result = [MSDBStorage executeSelectionQuery:@"PRAGMA auto_vacuum" inOpenedDatabase:db];
+  int vacuumMode = [(NSNumber *)result[0][0] intValue];
+  BOOL autoVacuumDisabled = vacuumMode != 1;
+
+  /*
+   * If `auto_vacuum` is disabled, change it to `FULL` and then manually `VACUUM` the database. Per the SQLite docs, changing the state of
```

Comment:
Can we link to the docs that are mentioned?

Reply:
+1